### ISSUE TITLE
This code need to use of a random proxy of a proxy list. otherwise blocked IP.

### ISSUE THREAD

**Issue by NabiKAZ** [llaera/slowloris.pl#2]
This code need to use of a random proxy of a proxy list. otherwise blocked IP.

DENY  xxxxxxxxx  *  in  57m 42s  lfd - *Port Scan* detected from xxxxxxxxx (US/United States/xxxxxxxxx-xxxxxx). 11 hits in the last 25 seconds

**Comment by Baldomo** [comment]
You can use Tor and route the traffic through its SOCKS proxy at port 9150 to get a reliable proxy that isn't easily traceable.

**Comment by NabiKAZ** [comment]
I don't talk about traceable. I talk about blocked IP for the reason many request in one time.
As you can see: `11 hits in the last 25 seconds`
My IP blocked after 25 sec.
I thinks need a mechanism for change IP automatically.

**Comment by Baldomo** [comment]
Changing ip automatically is not that straightforward. I'll work on that and will Try to keep you updated

**Comment by m4rc3l** [comment]
Agreed, no point starting an attack if IP isnt dynamical. Most hostig servers have DoS protection (i.e auto ip blocking).

**Comment by Baldomo** [comment]
If you want to dynamically change your IP, I believe Kali Linux has a module that can be summoned to do that - also randomizes your MAC address so it can cause network traffic problems, try to search a bit and keep an eye out.

**Comment by Beyarz** [comment]
I kinda agree with NabiKAZ.
